### DISCUSSION TITLE
Constructor of "VRFV2WrapperConsumerBase.sol" has 2 arguments instead of 1

### DISCUSSION THREAD

**Discussion by Ciphadus7** [Cyfrin/foundry-full-course-cu#454] [Q&A]
In the video, the constructor takes a single argument: address _vrfCoordinator.
In the updated code of VRFV2WrapperConsumerBase.sol, it takes two.
Namely:
  constructor(address _link, address _vrfV2Wrapper) { LINK = LinkTokenInterface(_link); VRF_V2_WRAPPER = VRFV2WrapperInterface(_vrfV2Wrapper); }

Stuck at this point.
Confused as to what it needs as the second argument.

**Comment by Niferu** [comment]
Hello @Ciphadus7 

`_vrfV2Wrapper` is the address of the VRFV2Wrapper contract. To understand this contract better visit [this](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/vrf/VRFV2WrapperConsumerBase.sol) page and check all developers notes above functions, constructor etc.
